code/modules/modular_computers/computers/machinery/modular_computer.dm

```diff
@@ -140,7 +140,7 @@
 	var/obj/item/stock_parts/cell/cell = get_cell()
 	if(isnull(cell) || cell.percent() >= 100)
 		return
-	var/power_to_draw = idle_power_usage * seconds_per_tick * 0.5
+	var/power_to_draw = idle_power_usage * seconds_per_tick * 0.5 // Why halve it?
```

Comment:
Because seconds_per_tick is by default 2 so halving it would become 1.

Actually by that logic you don't need seconds_per_tick at all

Reply:
Doesn't seem logical. The idle power usage should be halved instead I guess, so that it's actually truthful.